text/0000-sends-accepts.md

```diff
@@ -0,0 +1,46 @@
+- Start Date: 2015-12-3
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes adding `sends` and `accepts` attributes to the `Category` element
+and `Transaction` element in the API Description format.
+
+# Motivation
+
+This accommodates the ability to capture what mime types an API or Transaction
```

Comment:
Thinking more about this – I'd argue this should be nested under some sort of "mime-type" attribute.